inventory_management_system_api/routers/v1/usage_status.py

```diff
@@ -37,11 +39,19 @@
     status_code=status.HTTP_201_CREATED,
 )
 def create_usage_status(
-    usage_status: UsageStatusPostSchema, usage_status_service: UsageStatusServiceDep
+    usage_status: UsageStatusPostSchema, usage_status_service: UsageStatusServiceDep, request: Request
 ) -> UsageStatusSchema:
     logger.info("Creating a new usage status")
     logger.debug("Usage status data: %s", usage_status)
 
+    # check user is authorised to perform operation
+    if config.authentication.enabled is True:
+        jwt_bearer = JWTBearer()
+        if not jwt_bearer.is_jwt_access_token_authorised(request.state.token):
+            raise HTTPException(
+                status_code=status.HTTP_401_UNAUTHORIZED, detail="Not authorised to perform this operation"
```

Comment:
`401` is not the right code for this case. It is used when the user is not authenticated.